about.html

```diff
@@ -0,0 +1,178 @@
+        </div>
+        <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12 col-xl-12">
+          <div class="container">
+            <a href="">
+              <button type="button" class="btn my-bg-light btn-lg">
+                <div class="row">
+                  <div class="col-4">
+                    <img id="jambotronImg" src="assets/img/logo/Logo_final_transparent.png" alt="">
+                  </div>
+                  <div class="col-8 text-left">
+                    <h5 class="text-nowrap">
+                      Annal SMSE Summit
+                    </h5>
+                    <hr class="full-width">
+                    <h6 class="text-nowrap">2023.11.24(Fri) to 26(Sun)</h6>
+                    <hr class="full-width">
+                    <p class="text-nowrap">ICCB - Bamako(Mali)</p>
+                  </div>
+                </div>
+              </button>
+            </a>
+          </div>
+        </div>
+
+      </div>
+    </div>
+
+      <!-- ABOUT PAST EVENTS FOR PAGE 2-->
+      <div class="container space py-4">
+        <div class="container-fluid">
+          <div class="row text-center space">
+            <div class="col-12">
+              <h2>See past annual SMSE summits</h2>
+              <hr class="full-width mx-auto mb-5">
+              <div class="col-12">
+                <p>
+                  Take a look at the last 3 Annual summits.
+                </p>
+              </div>
+              
+            </div>
+          </div>
+          <div class="row text-center space">
+  
+            <div class="col-xs-12 col-sm-6 col-md-4 item1">
+              <img class="img-thumbnail img-fluid myMask" id="past-summits" src="assets/img/past-summits/2.jpg" alt="">
+              
+              <div class="past-summits-img-caption text-center card-img-overlay">
+                <h3 class="display-6 bold">2020</h3>
+                <h6 class="display-5 bold">ANNUAL SMSE SUMMIT 2020</h6>
+              </div>
+            </div>
+  
+            
+            <div class="col-xs-12 col-sm-6 col-md-4 item2">
+              <img class="img-thumbnail img-fluid myMask" id="past-summits" src="assets/img/past-summits/3.jpg" alt="">
+              <div class="past-summits-img-caption text-center card-img-overlay">
+                <h3 class="display-6 bold">2021</h3>
+                <h6 class="display-5 bold">ANNUAL SMSE SUMMIT 2021</h6>
+              </div>
+            </div>
+  
+            <div class="col-md-4 item3">
+              <img class="img-thumbnail img-fluid position-relative myMask" id="past-summits-images" src="assets/img/past-summits/4.jpg" alt="">
+              <div class="past-summits-img-caption text-center card-img-overlay">
+                <h3 class="display-6 bold">2022</h3>
+                <h6 class="display-5 bold">ANNUAL SMSE SUMMIT 2022</h6>
+              </div>
+            </div>        
+          </div>
+        </div>
+      </div> <!-- check if not an extra div -->
```

Comment:
- [ ] Kindly add only two pictures here to match what we have on the guiding template.

### Your Design
![your-pix-about-page](https://user-images.githubusercontent.com/26597432/235002158-30fc97b5-9959-4db8-86f5-58e940434b3e.png)

### Guiding Template
![temp-pix-about-page](https://user-images.githubusercontent.com/26597432/235002203-3f625289-6faa-4178-8ab7-bf4cf8d60869.png)

Reply:
Done @Qoosim, thank you for the comment

Reply:
Done @KayLemba. Added and after pseudo-classe. Thanks for the reminder..